tests/Builders/ServersBuilderTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vyuldashev\LaravelOpenApi\Tests\Builders;
+
+use Vyuldashev\LaravelOpenApi\Builders\ServersBuilder;
+use Vyuldashev\LaravelOpenApi\Tests\TestCase;
+
+class ServersBuilderTest extends TestCase
+{
+    /**
+     * @dataProvider providerBuild
+     * @param array $config
+     * @param array $expected
+     * @return void
+     */
+    public function testBuild(array $config, array $expected): void
+    {
+        $SUT = new ServersBuilder();
```

Comment:
what does `$SUT` mean?

Reply:
[System under test](https://en.wikipedia.org/wiki/System_under_test), it is the thing being tested.
I use this personally often as well, but mostly when building several mocked objects in a test, to make it more clear which of the objects you've created is the one actually being tested. 
This is not really needed in this test.